client/src/components/HeightClipper/HeightClipper.tsx

```diff
@@ -37,7 +40,26 @@ export class HeightClipper extends React.Component<
     };
     this.main = React.createRef();
     this.content = React.createRef();
+
+    this.debounced_mutation_callback = _.debounce(
+      (mutationList: MutationRecord[], observer: MutationObserver) => {
+        this.componentDidUpdate();
+      }
+    );
+
+    this.observer = new MutationObserver(this.debounced_mutation_callback);
+
+    this.observer.observe(document, {
```

Comment:
Using `document` as target node for now since calling a query selector returns a possible null type (which typescript won't accept)